vela/worker_test.go

```diff
@@ -300,6 +351,24 @@ func ExampleWorkerService_Add() {
 	fmt.Printf("Received response code %d, for worker %+v", resp.StatusCode, worker)
 }
 
+func ExampleWorkerService_RefreshAuth() {
```

Comment:
Where is this used? Same goes for the rest of the `ExampleWorkerService_*` funcs.

Reply:
honestly im not sure. i added it for consistency

Reply:
seemed like leftover boiler plate tests from the initial worker router commits. we can probably remove them in favor the more details tests above them

Reply:
this is used for documentation, see https://pkg.go.dev/github.com/go-vela/sdk-go@v0.18.1/vela#example-WorkerService.Update as an example which is extracted from `ExampleWorkerService_Update`

Reply:
oh!!! 😮‍💨 thats awesome. i didnt know go doc used test naming to populate that. thanks @wass3rw3rk